source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRIBaseCardElementRenderer.h

```diff
@@ -25,4 +25,5 @@
 @protocol ACRIKVONotificationHandler
 
 - (void)configUpdateForUIImageView:(ACOBaseCardElement *)acoElem config:(ACOHostConfig *)acoConfig image:(UIImage *)image imageView:(UIImageView *)imageView;
+
```

Comment:
Nit; looks like we have a stray newline added here-- probably just revert changes to this file altogether.